src/app/components/PointsCounter.js

```diff
@@ -0,0 +1,15 @@
+export class PointsCounter {
+    createPointsCounter(pkt = 0) {
+        const wrapper = document.createElement('div');
+        const desc = document.createElement('div');
+        const points = document.createElement('div');
+        desc.innerHTML = 'Points';
```

Comment:
Ja bym zamienił innerHTML na textContent lub innerText

Reply:
Dla dipów musisz najpierw stworzyć `textNode`, a dopiero później będziesz mógł zmienić jego zawartość.
```js
desc.appendChild(document.createTextNode('Points'))
```
Później jakbyś chciał update `textNode` zrobić, to możesz się odwołać właściwością `textContent`.

Reply:
#22 Feature